wallet/chain/p/wallet/backend_visitor.go

```diff
@@ -63,18 +67,47 @@ func (b *backendVisitor) RemoveSubnetValidatorTx(tx *txs.RemoveSubnetValidatorTx
 }
 
 func (b *backendVisitor) TransferSubnetOwnershipTx(tx *txs.TransferSubnetOwnershipTx) error {
-	b.b.setSubnetOwner(
+	b.b.setOwner(
 		tx.Subnet,
 		tx.Owner,
 	)
 	return b.baseTx(&tx.BaseTx)
 }
 
 func (b *backendVisitor) ConvertSubnetTx(tx *txs.ConvertSubnetTx) error {
+	for i, vdr := range tx.Validators {
+		b.b.setOwner(
+			tx.Subnet.Prefix(uint64(i)),
+			&secp256k1fx.OutputOwners{
+				Threshold: vdr.DeactivationOwner.Threshold,
+				Addrs:     vdr.DeactivationOwner.Addresses,
+			},
+		)
+	}
 	return b.baseTx(&tx.BaseTx)
 }
 
 func (b *backendVisitor) RegisterSubnetValidatorTx(tx *txs.RegisterSubnetValidatorTx) error {
+	warpMessage, err := warp.ParseMessage(tx.Message)
```

Comment:
where do we validate/verify the Warp message?

Reply:
In the wallet, transactions aren't verified (it is assumed that the API the wallet is interacting with is virtuous).

At this point in the wallet code, the API has told us that the transaction was confirmed.

As far as where Warp message verification happens in the node, it is added in this PR: #3423